.flowconfig

```diff
@@ -3,3 +3,5 @@
 
 [options]
 suppress_comment= \\(.\\|\n\\)*\\$FlowIssue
+
+unsafe.enable_getters_and_setters=true
```

Comment:
I filed an Issue against FlowType to see about `WeakMap` safety: https://github.com/facebook/flow/issues/2742